### PR TITLE
Update turn-on-or-off-skype-for-business-client-feedback-reporting.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-skype-for-business-client-feedback-reporting.md [hunk 1]
```diff
@@ -57,11 +57,12 @@ If you need to know more, see [Connect to all Microsoft 365 or Office 365 servic
     
 2. In the **Windows PowerShell** window, connect to your Microsoft 365 or Office 365 by running:
     
-    > [!NOTE]
-    > You only have to run the **Import-Module** command the first time you use the Skype for Business Online Windows PowerShell module.
+  > [!NOTE]
+  > Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-skype-for-business-client-feedback-reporting.md:61]:
```suggestion
  > The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**